linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/ansible/shared.yml

```diff
@@ -12,7 +12,7 @@
 - name: "Set fact: Package manager reinstall command (yum)"
   set_fact:
     package_manager_reinstall_cmd: yum reinstall -y
-  when: (ansible_distribution == "RedHat" or ansible_distribution == "OracleLinux")
+  when: (ansible_distribution == "RedHat" or ansible_distribution == "CentOS" or ansible_distribution == "OracleLinux")
```

Comment:
Could you update the `platform` header in this Ansible remediation to also include RHEL9, please? I think that `multi_platform_rhel` would be fine here.

Reply:
Updated